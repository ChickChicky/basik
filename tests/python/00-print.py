print("Hello, world !")
print("Hello,","world !")
print("Hello,","world","!")
print(10)
print(284525)