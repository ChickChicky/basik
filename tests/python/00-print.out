Hello, world !
Hello, world !
Hello, world !
10
284525
